CMakeLists.txt

```diff
@@ -44,7 +44,11 @@ if (AALWINES_GetDependencies)
         CONFIGURE_COMMAND ""
         INSTALL_COMMAND cd ../rapidxml-ext && ${CMAKE_COMMAND} -E copy rapidxml.hpp rapidxml_iterators.hpp rapidxml_print.hpp rapidxml_utils.hpp ${EXTERNAL_INSTALL_LOCATION}/include 
     )
-
+    file(DOWNLOAD
```

Comment:
What is the purpose of this fix?

Reply:
Or rather, is this sufficient to get the full json functionality without the binary blowing up in size?

Reply:
This fetches the [nlohmann::json](https://github.com/nlohmann/json) library, which I use to parse the weight file. The header file downloaded (json.hpp) contains the entire library. 
I chose this library since it is easy to use.